init/redhat/waagent.service

```diff
@@ -8,9 +8,11 @@ ConditionPathExists=/etc/waagent.conf
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/python3 -u /usr/sbin/waagent -daemon
+ExecStart=/usr/bin/python -u /usr/sbin/waagent -daemon
```

Comment:
why this change?

Reply:
I explained in other comment.

Reply:
since this the "main" file, I'd leave the Python version as 3 here, and create a py2 directory for the older distros (instead of py3)

Reply:
updated